p_config.json

```diff
@@ -15,6 +15,8 @@
     "4.14 heavy build": "periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-heavy-build-ovn-remote-libvirt-ppc64le",
     "4.15 libvirt": "periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-remote-libvirt-ppc64le",
     "4.15 powervs": "periodic-ci-openshift-multiarch-master-nightly-4.15-ocp-e2e-ovn-ppc64le-powervs-per",
```

Comment:
please remove this. Also did you check how this new two job links will work for job history ? Shouldn't we change the code there as well to adapt this ?

Reply:
Done, The new links work fine with job history, after adding new ci links in p_config.json we don't require any code changes to be done in CI_JobHistory.py script.